README.md

```diff
@@ -1572,6 +1572,7 @@ _Related: [Booking and Scheduling](#booking-and-scheduling)_
 
 - [Bitpoll](https://github.com/fsinfuhh/Bitpoll) - A web application for scheduling meetings and general polling. ([Demo](https://bitpoll.mafiasi.de/)) `GPL-3.0` `Python`
 - [Calagator](https://calagator.org/) - Event aggregator. ([Source Code](https://github.com/calagator/calagator)) `MIT` `Ruby`
+- [Claper](https://claper.co/) - The ultimate tool to interact with your audience. An open-source alternative to Slido, AhaSlides and Mentimeter. `GPL-3.0` `Elixir`
```

Comment:
- Please add the source code: https://github.com/ClaperCo/Claper

> https://github.com/ClaperCo/Claper#getting-started

- Please change the language to `Docker/Nodejs/Shell`

Reply:
Hi Kieran, 

I don't understand why I should change to `Docker/Nodejs/Shell` because this is an Elixir project. NodeJS is only required for npm to run for the Javascript side and it's not on the checklist: 

> The Language tag is the main server-side requirement for the software. Don't include frameworks or specific dialects.

There is just small Shell scripts but not required. The checklist says `main server-side requirement` and Elixir is the main server-side requirement.

There is a Docker image available so I can change to `Elixir/Docker` it will be more suitable.

Reply:
According to https://github.com/ClaperCo/Claper#installation, For Installation, Shell and Nodejs are required to install the software. If these are optional, then feel free to leave them off however if they are required for installation,  please change the language

Reply:
Shell and nNodejs are required for local development not for production environment that's why ;)